installer/bundle/bundle_skel.sh

```diff
@@ -573,13 +572,6 @@ if [ "$ONBOARD_ERROR" -ne 0 ]; then
     exit 1
 fi
 
-if [ "$onboardINT" -ne 0 ]; then
-    if [ -z "$onboardID" -o -z "$onboardKey" ]; then
```

Comment:
add this check below, since if -d was specified it means they want to onboard and must specify the id and key